### DISCUSSION TITLE
Downtime for https://search.sapti.me

### DISCUSSION THREAD

**Discussion by samsapti** [searxng/searx-instances#472] [Report instance downtime]
Short downtime will occur due to migration to a new VPS.

**Comment by samsapti** [comment]
Instance is up and running again.

**Comment by unixfox** [comment]
For very short downtimes it's OK to not tell us.

It becomes an issue when it's more than 6 hours for example.

But I appreciate the dedication to tell us.

**Reply by samsapti** [reply]
Thanks for letting me know, I was not aware. I thought it would be nice for the users as well for the notice to be here so they know what's going on.
